docs/tutorials/own_machine/05_start_renku_session.rst

```diff
@@ -0,0 +1,81 @@
+.. _start_renku_session:
+
+Start a Renku session
+=====================
+
+#. **Navigate into your renku project directory:**
+
+   .. code-block:: shell-session
+
+       $ cd <project_name>
+
+
+#.  **Start a renku session:**
+
+    .. code-block:: shell-session
+
+       $ renku session start
+       ID          STATUS    URL
+       ----------  --------  ------------------------------------------------------------
+       f1693c198e  running   http://0.0.0.0:56674/?token=910ca732ef574049a22d41d0f1109f56
+
+    .. note::
+
+       Note: Renku may inform you that the container image for the current commit
+       does not yet exist on your machine, and Renku will offer to build it for you. Say yes!
+
+    .. note::
+
+        **Building too many images?**
+        By default, Renku builds a new image every time you start from a new commit.
+        If you would prefer to reuse an image already built for your project (not build a new image every time),
+        you can :ref:`pin a specific Docker image <pin_docker_image>` that every session will use.
+
+
+#.  **Open the session in your browser.**
+    When the session starts, it will print out a url where the session is running.
+    Copy this url into your browser to access your Renku project running inside it's containerized environment.
+
+    If you need to find this url again later, you can find all running Renku sessions by running `renku session list`.
+
+    .. code-block:: shell-session
+
+        $ renku session list
+        ID          STATUS    URL
+        ----------  --------  ------------------------------------------------------------
+        f1693c198e  running   http://0.0.0.0:56674/?token=910ca732ef574049a22d41d0f1109f56
+
+    You can also use the session ID to call `renku session open <ID>`, which opens your browser window for you.
+
+    .. code-block:: shell-session
+
+        $ renku session open <session_id>
+
+.. note::
+
+    **Looking for a shell?**
+
+    Would you like to enter your containerized project environment on a shell, rather than via the browser?
+
+    Since Renku uses Docker to manage your project's computational environment, you can use Docker commands to enter the
+    container directly and use the shell.
+
+
+    First, find your renku session's container ID by listing your running sessions:
+
+    .. code-block:: shell-session
+
+       $ renku session list
+       ID          STATUS    URL
+       ----------  --------  ------------------------------------------------------------
+       f1693c198e  running   http://0.0.0.0:56674/?token=910ca732ef574049a22d41d0f1109f56
+
+
+    Note the value in the ID field.
+
+    Then, open a shell in a running container by providing the Container ID:
+
+    .. code-block:: console
+
+        $ docker exec -it <ID> /bin/bash
+        base ▶ ~ ▶ work ❯ project_name ▶ master ▶ $ ▶
```

Comment:
Would it make sense to add instructions to stop the renku session and maybe prune accumulating images ? here (or just link to the official [docs](https://docs.docker.com/config/pruning/)) ?

Reply:
Oh yes excellent point! I added `docker image prune -a --filter "until=24h"` and `docker container prune --filter "until=24h"`. Would you recommend anything additional/different?

Reply:
Sounds good !